ui.h

```diff
@@ -97,6 +97,8 @@ _UI_EXTERN void uiControlHide(uiControl *);
 _UI_EXTERN int uiControlEnabled(uiControl *);
 _UI_EXTERN void uiControlEnable(uiControl *);
 _UI_EXTERN void uiControlDisable(uiControl *);
+_UI_EXTERN void uiSize(uiControl *control, int *width, int *height);
+_UI_EXTERN void uiSetSize(uiControl *control, int width, int height);
```

Comment:
For consistency, should these be names uiControlSize and uiControlSetSize, respectively?